.travis.yml

```diff
@@ -1,6 +1,12 @@
 language: c
 install: wget https://raw.githubusercontent.com/ocaml/ocaml-travisci-skeleton/master/.travis-opam.sh
-script: bash -ex .travis-opam.sh
+script:
+  - cp xapi-inventory.opam opem && bash -ex .travis-opam.sh
```

Comment:
`opem` -> `opam`

Reply:
But I think this will not work. I think you should add the following to global:
```
- PINS: "xcp-inventory:. xapi-inventory:."
```
and not copy any file